pkg/parser/keywords.go

```diff
@@ -0,0 +1,626 @@
+	{"IMPORTS", false},
+	{"INCREMENT", false},
+	{"INCREMENTAL", false},
+	{"INDEXES", false},
+	{"INSERT_METHOD", false},
+	{"INSTANCE", false},
+	{"INVISIBLE", false},
+	{"INVOKER", false},
+	{"IO", false},
+	{"IPC", false},
+	{"ISOLATION", false},
+	{"ISSUER", false},
+	{"JSON", false},
+	{"KEY_BLOCK_SIZE", false},
+	{"LABELS", false},
+	{"LANGUAGE", false},
+	{"LAST", false},
+	{"LAST_BACKUP", false},
+	{"LASTVAL", false},
+	{"LESS", false},
+	{"LEVEL", false},
+	{"LIST", false},
+	{"LOCAL", false},
+	{"LOCAL_ONLY", false},
+	{"LOCKED", false},
+	{"LOCATION", false},
+	{"LOGS", false},
+	{"MASTER", false},
+	{"MAX_IDXNUM", false},
+	{"MAX_MINUTES", false},
+	{"MAX_CONNECTIONS_PER_HOUR", false},
+	{"MAX_QUERIES_PER_HOUR", false},
+	{"MAX_ROWS", false},
+	{"MAX_UPDATES_PER_HOUR", false},
+	{"MAX_USER_CONNECTIONS", false},
+	{"MB", false},
+	{"MEMBER", false},
+	{"MEMORY", false},
+	{"MERGE", false},
+	{"MICROSECOND", false},
+	{"MIN_ROWS", false},
+	{"MINUTE", false},
+	{"MINVALUE", false},
+	{"MODE", false},
+	{"MODIFY", false},
+	{"MONTH", false},
+	{"NAMES", false},
+	{"NATIONAL", false},
+	{"NCHAR", false},
+	{"NEVER", false},
+	{"NEXT", false},
+	{"NEXTVAL", false},
+	{"NO", false},
+	{"NOCACHE", false},
+	{"NOCYCLE", false},
+	{"NODEGROUP", false},
+	{"NOMAXVALUE", false},
+	{"NOMINVALUE", false},
+	{"NONCLUSTERED", false},
+	{"NONE", false},
+	{"NOWAIT", false},
+	{"NVARCHAR", false},
+	{"NULLS", false},
+	{"OFF", false},
+	{"OFFSET", false},
+	{"OLTP_READ_ONLY", false},
+	{"OLTP_READ_WRITE", false},
+	{"OLTP_WRITE_ONLY", false},
+	{"TPCH_10", false},
+	{"ON_DUPLICATE", false},
+	{"ONLINE", false},
+	{"ONLY", false},
+	{"OPEN", false},
+	{"OPTIONAL", false},
+	{"PACK_KEYS", false},
+	{"PAGE", false},
+	{"PARSER", false},
+	{"PARTIAL", false},
+	{"PARTITIONING", false},
+	{"PARTITIONS", false},
+	{"PASSWORD", false},
+	{"PAUSE", false},
+	{"PERCENT", false},
+	{"PER_DB", false},
+	{"PER_TABLE", false},
+	{"PLUGINS", false},
+	{"POINT", false},
+	{"POLICY", false},
+	{"PRE_SPLIT_REGIONS", false},
+	{"PRECEDING", false},
+	{"PREPARE", false},
+	{"PRESERVE", false},
+	{"PRIVILEGES", false},
+	{"PROCESS", false},
+	{"PROCESSLIST", false},
+	{"PROFILE", false},
+	{"PROFILES", false},
+	{"PROXY", false},
+	{"PURGE", false},
+	{"QUARTER", false},
+	{"QUERIES", false},
+	{"QUERY", false},
+	{"QUICK", false},
+	{"RATE_LIMIT", false},
+	{"REBUILD", false},
+	{"RECOVER", false},
+	{"REDUNDANT", false},
+	{"RELOAD", false},
+	{"REMOVE", false},
+	{"REORGANIZE", false},
+	{"REPAIR", false},
+	{"REPEATABLE", false},
+	{"REPLICA", false},
+	{"REPLICAS", false},
+	{"REPLICATION", false},
+	{"REQUIRED", false},
+	{"RESOURCE", false},
+	{"RESPECT", false},
+	{"RESTART", false},
+	{"RESTORE", false},
+	{"RESTORES", false},
+	{"RESUME", false},
+	{"REUSE", false},
+	{"REVERSE", false},
+	{"ROLE", false},
+	{"ROLLBACK", false},
+	{"ROLLUP", false},
+	{"ROUTINE", false},
+	{"ROW_COUNT", false},
+	{"ROW_FORMAT", false},
+	{"RTREE", false},
+	{"HYPO", false},
```

Comment:
Should we sort this first? Or just fix the parser.y file?

Reply:
If the table needs sorting then `ORDER BY` is the best solution. If the entries in this file need to be in alphabetical order, then fixing this in `parser.y` is probably best.

Reply:
https://github.com/pingcap/tidb/pull/48851